src/components/NestedScrollView.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {ScrollView, View} from 'react-native';
+import styles from '../styles/styles';
+
+const propTypes = {
+    /** Any children to display */
+    children: PropTypes.node,
```

Comment:
Will this component have any use without children?

Reply:
Ummm, I don't think so.

Reply:
so shouldn't this be required?